layouts/shortcodes/tabs.html

```diff
@@ -0,0 +1,12 @@
+<div id="tabs-with-content">
+  <div class="tabs is-left is-toggle">
+    <ul>
+      {{ range $index, $value := .Params }}
+      <li><a href="#{{$index}}">{{$value}}</a></li>
```

Comment:
I think this is what the build is complaining about -- whatever anchor this is pointing to doesn't exist.